modules/cnf-topology-aware-lifecycle-operator-installation-web-console.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-talo-for-cluster-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="installing-topology-aware-lifecycle-operator-using-web-console_{context}"]
+= Installing the Topology Aware Lifecycle Operator by using the web console
+
+You can use the {product-title} web console to install the Topology Aware Lifecycle Operator.
+
+.Prerequisites
+
+// Based on polarion test cases
+
+* Install the latest version of {rh-rhacm} Operator.
+* Set up a hub cluster with disconnected regitry.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Operators* -> *OperatorHub*.
+. Search for the *Topology Aware Lifecycle Operator* from the list of available Operators, and then click *Install*.
+. Keep the default selection of *Installation mode* and *namespace* to ensure that the Operator is installed in the `openshift-cluster-group-upgrades` namespace.
+. Click *Install*.
+
+.Verification
+
+To confirm that the installation is successful:
+
+. Navigate to the *Operators* -> *Installed Operators* page.
+. Check that the Operator is installed in the `openshift-cluster-group-upgrades` namespace and its status is `Succeeded`.
```

Comment:
Check that the Operator is installed in `All Namespaces` and its status is `Succeeded`.